CONTRIBUTING.md

```diff
@@ -19,6 +19,14 @@ Or the linter:
 $ npm run lint
 ```
 
+If you want to run a single unit test, you can globally install the test runner
+and point it at the file you're interested in
+
+```sh
+$ npm i -g mocha
+$ mocha -t 5000 path/to/your/test.js
```

Comment:
It's kind of dangerous to install mocha in such way as you might end up with version different from what `lerna` uses? It also leads to pollution of global namespace. As `mocha` is already part of `devDependencies` and aliased via `test` script, you can do the following:

```bash
npm run test -- <path> # or any other cmd arg mocha understands
```